examples/ScanLoop.cpp

```diff
@@ -0,0 +1,65 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// Copyright 2020 Lawrence Livermore National Security, LLC and other CARE developers.
+// See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//////////////////////////////////////////////////////////////////////////////////////
+
+// This macro needs to be defined before including care/care.h,
+// which allows you to port a file at a time. Without this define,
+// LOOP_STREAM will run on the CPU. With this define, LOOP_STREAM
+// will run on the GPU only if you built with CUDA enabled. Otherwise,
+// if CUDA is disabled, it will default back to running on the host.
+#define GPU_ACTIVE
+
+// CARE library header
+#include "care/care.h"
+#include "care/scan.h"
+
+// Std library headers
+#include <iostream>
+#include <vector>
+
+int main(int, char**) {
+   // Normal code
+   int size = 1000;
+   std::vector<int> data;
+
+   int threshold = 420;
+
+   for (int i = 0; i < size; ++i) {
+      int value = i * i;
```

Comment:
Since this is under examples, I would be very generous with comments. Make comments as if you are explaining to code to someone who has never seen RAJA or care .

Reply:
Done.